apps/site/lib/site_web/controllers/schedule/cms.ex

```diff
@@ -19,6 +19,35 @@ defmodule SiteWeb.ScheduleController.CMS do
   @impl Plug
   def init([]), do: []
 
+  @spec get_mode_from_route_description(Route.gtfs_route_desc()) :: String.t() | nil
+  defp get_mode_from_route_description(description) do
```

Comment:
This is a real nitpick, but could you please move this function lower? In Elixir, I'm a big adherent of the convention of putting public functions at the top of the file and the private functions below. Personally, I find it helps make the file easier to read: I can see the general things the module does quickly, and only dig down into the details if I'm interested.